### PR TITLE
docs: allcontributors description as issue template + description in docu

### LINKED ISSUE #191: Write a small guideline for allcontributors-Bot
We should write a small guideline for using the bot because it is easy to run into bugs and conflicts.

### PR DESCRIPTION
**Pull request by mschwrdtnr**
I don't think that it is necessary to write something about the bot into the documentation. Because of that i added a small issue-template with an usage-description.

This will closes #191.

View the example under https://github.com/mschwrdtnr/phonebook/issues/new/choose

### CODE REVIEW

**Comment by DanielHabenicht** [comment]
Preview Environment ready at https://13.79.231.25

**Comment by DanielHabenicht** [comment]
Preview Environment ready at https://13.79.231.25

**Comment by DanielHabenicht** [comment]
It might be good to have a page or a link to the bot nevertheless? Maybe even in the Contributing.md as last step. 
`6. Add yourself as a Contributer by ...`

Can anybody use the bot or only members of the org?

**Comment by mschwrdtnr** [comment]
> It might be good to have a page or a link to the bot nevertheless? Maybe even in the Contributing.md as last step.
> `6. Add yourself as a Contributer by ...`

You mean [here](https://t-systems-mms.github.io/phonebook/pages/development-guides/)?

> Can anybody use the bot or only members of the org?

I am not 100% sure, but i think that everyone can use the bot. I don't get a thing why this should not work.

**Comment by DanielHabenicht** [comment]
Preview Environment ready at https://13.79.231.25

File: docs/pages/development-guides/general-guide.md [hunk 1]
```diff
@@ -32,16 +33,16 @@ This is the current project structure.
...
 4. Create a Pull Request from your Branch.
 5. Wait for approval from one of the maintainers.
+6. **Extra:** If you're not listed for the type of contribution you can add yourself as a contributor with the [@all-contributors-Bot](https://allcontributors.org/docs/en/overview) by writing `@all-contributors please add @username for` [contribution](https://allcontributors.org/docs/en/emoji-key) in your PR as a comment.
 
 > If you need a tutorial about important git commands look [here](https://dev.to/dhruv/essential-git-commands-every-developer-should-know-2fl).
 
 ---
 
-We splitted the development documentation in two different parts.
+We have splitted the development documentation in two different parts.
```
Comment (DanielHabenicht) [inline docs/pages/development-guides/general-guide.md]:
```suggestion
We split the development documentation in two different parts.
```

**Pull request merged**